packages/language-server/src/core/documents/DocumentMapper.ts

```diff
@@ -143,6 +143,7 @@ export class SourceMapDocumentMapper implements DocumentMapper {
 		}
 
 		if (mapped.line === 0) {
+			// eslint-disable-next-line no-console
```

Comment:
I normally wouldn't do this (especially since the console log is mostly a debug thing) however, that part of the file will get rewritten soon when the new TSX output will be added, so just for now it makes ESLint happy